src/frontend/src/lib/i18n/en.json

```diff
@@ -275,7 +275,8 @@
 			"hide_zero_balances": "Hide zero balances",
 			"all_tokens_with_zero_hidden": "All tokens with zero balance are currently hidden.",
 			"initializing": "Initializing...",
-			"updating_ui": "Updating the UI..."
+			"updating_ui": "Updating the UI...",
+			"show_hide_token": "Show or hide token"
```

Comment:
Shouldn't we replace globally `hide_zero_balances` by `show_hide_token` for UX consistency?

Reply:
the only "incorrect" usage I found was here, do you see it somewhere else?

Reply:
Ah now I get it.